From patches Mon Sep 17 00:00:00 2001
From: Stefan Hajnoczi <stefanha@redhat.com>
To: qemu-devel@nongnu.org
Cc: Anthony Liguori <aliguori@us.ibm.com>,
 Stefan Hajnoczi <stefanha@redhat.com>,
 Peter Crosthwaite <peter.crosthwaite@xilinx.com>
Date: Fri, 19 Apr 2013 13:34:01 +0200
Message-id: 1366371241-23430-9-git-send-email-stefanha@redhat.com
Subject: [PATCH 8/8] m25p80: Remove bogus include of devices.h

From: Peter Crosthwaite <peter.crosthwaite@xilinx.com>

I think in the early revisions of this we had an instantiation helper
for the device in devices.h. This was later removed and this header was
left over. Removed

Signed-off-by: Peter Crosthwaite <peter.crosthwaite@xilinx.com>
Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>
---
 hw/block/m25p80.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/hw/block/m25p80.c b/hw/block/m25p80.c
index efcc7f4..b3ca19a 100644
--- a/hw/block/m25p80.c
+++ b/hw/block/m25p80.c
@@ -24,7 +24,6 @@
 #include "hw/hw.h"
 #include "sysemu/blockdev.h"
 #include "hw/ssi.h"
-#include "hw/devices.h"
 
 #ifndef M25P80_ERR_DEBUG
 #define M25P80_ERR_DEBUG 0
-- 
1.8.1.4

